src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -630,7 +635,27 @@ private void getCanvasResponse(HttpServletRequest request,
 		}else if(request.getMethod().equals(DELETE)) {
 			clientRequest=new HttpDelete(url);
 		}
-		BufferedReader rd = processApiCall(clientRequest);
+
+		HttpResponse canvasResponse = processApiCall(clientRequest);
+		BufferedReader rd = new BufferedReader(new InputStreamReader(canvasResponse.getEntity().getContent()));
+
+		String linkValue = null;
+		
+		Header[] headers = canvasResponse.getAllHeaders();
+		for (Header header : headers) {
+			M_log.debug("Key : " + header.getName() 
+			      + " ,Value : " + header.getValue());
+			if(header.getName().equals("Link")){
+				linkValue = header.getValue();
```

Comment:
Can you break here?  That makes it clear that you are only looking for 1 values.  

If you need to get another specific header, consider making this a utility method.